base/sg_cluster_cfg.go

```diff
@@ -128,7 +133,10 @@ func (c *CfgSG) Subscribe(cfgKey string, ch chan cbgt.CfgEvent) error {
 }
 
 func (c *CfgSG) FireEvent(docID string, cas uint64, err error) {
-
+	if c.groupID != nil {
+		// Remove first instance of group ID
+		docID = strings.Replace(docID, *c.groupID, "", 1)
```

Comment:
It would be preferable to define the combined prefix at CfgSG initialization time and just use that with strings.TrimPrefix below, instead of doing multiple string operations.